pkgs/development/python-modules/devpi-ldap/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  setuptools,
+  devpi-server,
+  pyyaml,
+  ldap3
+}:
+
+buildPythonPackage rec {
+  pname = "devpi-ldap";
+  version = "2.1.1";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-UhMWCwtXQTLg/rGYm9pvWy8BXo6fR24ZgyTmoJG6nrg=";
+  };
```

Comment:
please use fetchFromGitHub or GitLab or wherever the source code it. It is preferred now.

Reply:
sure, was just following what the other devpi-* packages do